### PR TITLE
Added organisation setup routes, refactored helper functions

### PR DESCRIPTION
**Pull request by Saumya29**

### CODE REVIEW

File: src/helpers/handleHash.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import bcrypt from 'bcrypt';
+
+import { productionConstants } from '../config/constants';
+
+const generateHash = async (plainText) => {
+  const hash = await bcrypt.hash(plainText, (+productionConstants.SALT_ROUNDS));
```
[the remark this answers is not in the document]
Comment (Saumya29) [reply src/helpers/handleHash.js]: Converting string to number

[the remark this answers is not in the document]
Comment (anshurathee) [reply src/helpers/handleHash.js]: Use Number()

File: src/routes/dashboard.js [hunk 2]
```diff
@@ -59,4 +79,36 @@ route.post('/setup', async (req, res) => {
...
+      },
+    });
+  }
+  console.log(ownerID);
+  const response = await registerOrganisation(
+    db,
+    organisationCollection,
+    {
+      ownerID,
+      ...req.body,
```
Comment (anshurathee) [inline src/routes/dashboard.js]: spread it before, and then pass ownerID

File: src/routes/dashboard.js [hunk 3]
```diff
@@ -59,4 +79,36 @@ route.post('/setup', async (req, res) => {
   res.json(response);
 });
 
+// For setting up the organisation details
+route.post('/orgSetup', async (req, res) => {
```
Comment (anshurathee) [inline src/routes/dashboard.js]: use route constants

File: src/routes/dashboard.js [hunk 4]
```diff
@@ -48,7 +69,6 @@ route.get('/', (req, res) => {
 route.post('/setup', async (req, res) => {
```
Comment (anshurathee) [inline src/routes/dashboard.js]: use route constants

File: src/routes/dashboard.js [hunk 5]
```diff
@@ -40,6 +41,26 @@ export const updateUser = async (db, collection, email, dataToBeUpdated) => {
...
+
+//   } catch (e) {
+//     return {
+//       errors: {
+//         name: '[Register Org]: Caught an error while creating the organisation',
+//       },
+//     };
+//   }
+// };
 route.get('/', (req, res) => {
```
Comment (anshurathee) [inline src/routes/dashboard.js]: use route constants

File: src/routes/dashboard.js [hunk 6]
```diff
@@ -40,6 +41,26 @@ export const updateUser = async (db, collection, email, dataToBeUpdated) => {
   }
 };
 
+// export const addOrganisation = async (
```
Comment (anshurathee) [inline src/routes/dashboard.js]: remove the commented out code.

**Review by anshurathee — CHANGES_REQUESTED** [review]

File: src/routes/dashboard.js (hunk 4 above)
Comment (Saumya29) [reply src/routes/dashboard.js]: Done

File: src/routes/dashboard.js (hunk 5 above)
Comment (Saumya29) [reply src/routes/dashboard.js]: Done

File: src/routes/dashboard.js (hunk 6 above)
Comment (Saumya29) [reply src/routes/dashboard.js]: Done

File: src/routes/dashboard.js (hunk 4 above)
Comment (Saumya29) [reply src/routes/dashboard.js]: Done

**Review by anshurathee — APPROVED** [review]

**Pull request merged**